### ISSUE TITLE
Misleading error if missing comma in object literal

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#467]
```js
let o = {
  m: async () => {
  }  // line 3: missing comma
  ["g"]: async () => {
  }
}
```

qljs should tell us that we're missing a comma on line 3, but it instead complains about the `:` on line 4 (then crashes):
```
hello.js:4:8: error: missing comma between object literal entries [E025]
```

**Comment by strager** [comment]
This is already captured in #277.

**Comment by strager** [comment]
Actually, this is a different case from #277. Specifically, this issue happens with computed property names.
